### ISSUE TITLE
BLOsc error build and install on Linux

### ISSUE THREAD

**Issue by duckt14** [madskjeldgaard/portedplugins#39]
Hi,
I'm unable to install the plugins because CMake tells me that BLOsc was not declared during the building.
This is the text of the error:

`In file included from /home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:4:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.hpp:26:18: error: ‘BlOsc’ in namespace ‘daisysp’ does not name a type
   26 |         daisysp::BlOsc blosc;
      |                  ^~~~~
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp: In constructor ‘BLOsc::BLOsc::BLOsc()’:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:14:3: error: ‘blosc’ was not declared in this scope
   14 |   blosc.Init(samplerate);
      |   ^~~~~
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp: In member function ‘void BLOsc::BLOsc::next(int)’:
/home/enrico/portedplugins/plugins/BLOsc/BLOsc.cpp:32:5: error: ‘blosc’ was not declared in this scope
   32 |     blosc.SetFreq(slopedFreq.consume());
      |     ^~~~~
gmake[2]: *** [CMakeFiles/BLOsc_scsynth.dir/build.make:76: CMakeFiles/BLOsc_scsynth.dir/plugins/BLOsc/BLOsc.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1898: CMakeFiles/BLOsc_scsynth.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2`

Any solutions?
Thank you, Enrico :)

**Desktop:**
 - OS: Fedora Linux 39
 - Architecture: x86
 - SC Version: 3.14.0-dev
 - PortedPlugins Version: v0.4.1

**Comment by WhiteGobo** [comment]
I have the same error on arch linux via `yay -S supercollider-portedplugins `.
daisysp-library updated on january 8 to v1.0.
The `Source/Synthesis/blosc.h/cpp` were removed in the new version

**Comment by WhiteGobo** [comment]
They moved faust modules to a new submodule. [See here for more information](https://github.com/electro-smith/DaisySP/pull/196)
They said, "after updating libDaisy, and DaisySP (including pulling the new submodule), users can add USE_DAISYSP_LGPL=1 to their Makefile, and continue without any other changes."

**Comment by WhiteGobo** [comment]
With the PR the old version of daisysp will be used. That fixes the problem for me.
Adding the option USE_DAISYSP_LGPL should fix the problem also. So adding

```
target_compile_definitions("${plugin_name}_scsynth PUBLIC "USE_DAISYSP_LGPL")
```
after:

https://github.com/madskjeldgaard/portedplugins/blob/24639cdeb48441e52c493de54aa822355326869a/CMakeLists.txt#L97

should fix the problem. But cmake of daisysp doesnt load daisysp-lgpl automaticly and i didnt got it to work.

I tried to fix the problem with the current version of daisysp but i guess daisysp-lgpl should be loaded automaticly from daisysp, and that isnt the case currently. So I tried adding

```
        CPMAddPackage(
                NAME daisysplgpl
                GITHUB_REPOSITORY "electro-smith/DaisySP-LPGL"
                GIT_TAG master
                )
```
after 

https://github.com/madskjeldgaard/portedplugins/blob/24639cdeb48441e52c493de54aa822355326869a/CMakeLists.txt#L670-L680

and adding after setting USE_DAISYSP_LGPL:

```
    target_include_directories("${plugin_name}_scsynth" PUBLIC include/daisysplgpl)
    target_link_libraries("${plugin_name}_scsynth" DaisySP-LGPL)
```

Then i got an error that within daisysplpgl dsp.h from daisysp couldnt be found.
